Weather Alert/Model.xcdatamodeld/Model.xcdatamodel/contents.orig

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
This file also needs to be removed... can you please go to your project folder and remove all .orig files? You can see the paths for them in this release, so to some you can navigate in terminal using `cd` command and `rm` command to remove